### ISSUE TITLE
Accessibility: Select-all and filter dropdown have no label

### ISSUE THREAD

**Issue by sedlatschek** [laravel/nova-issues#3392]
- Laravel Version: 8.41.0
- Nova Version: 3.25.0
- PHP Version: 7.4.6
- Database Driver & Version: MySQL 5.7
- Operating System and Version: Win10
- Browser type and version: Chrome 88

### Description:

It seems like dropdown triggers are problematic for accessibility. For example: [JAWS 2021](https://www.freedomscientific.com/products/software/jaws/) picks up the `select-all-dropdown` as _unlabeled control_. For the `filter-selector` dropdown trigger it says _unlabled control 0_ (or whatever amount of filters are active). The lens dropdown trigger on the other hand works as expected and is read as _Lenses_.

Please add a title attribute or an `sr-only` element to dropdown-triggers to have screenreaders read something that is more declarative.

![dropdown triggers](https://user-images.githubusercontent.com/58665536/118524388-624cfc80-b73e-11eb-99bf-bc9371f76c47.png)

Thank you

**Comment by davidhemphill** [comment]
👋 Hi there! In order to keep this repository focused on bug reports, we auto-close feature requests and requests for help. Feel free to post your feature requests so others can discuss and add reactions. We'll keep an eye on them for later planning.
